NodeJS/__unit__/AgentClient.test.ts

```diff
@@ -13,19 +11,21 @@ describe("AgentClient.ts", () => {
     });
     test("createAgentAndStation throws an error given an invalid password", async () => {
         expect.assertions(1);
-        let skillsWithPriorities = [{ "skillNumber": 100, "skillPriority": 5 }]
-        await expect( client.createAgentAndStation("agent1", "badpassword", skillsWithPriorities)).rejects.toEqual("invalid password")
+        await expect( client.createAgentAndStation("agent1", "badpassword")).rejects.toEqual("invalid password")
```

Comment:
nit: extra space after `expect(`

VSCode can autofix these types of issues by hitting the Code Format button combination:

```
On Windows Shift + Alt + F.
On Mac Shift + Option + F.
```

Reply:
Need to find a way to do it automatically.

Reply:
Ian installed the NPM package husky ([github link](https://github.com/typicode/husky)) which runs commands on based on git commands (for instance, you can have husky run auto formatting on each git commit). It's amazing and is pretty quick to setup.

Reply:
Definitely worth checking it out later.